scout/commands/update/genes.py

```diff
@@ -133,12 +133,12 @@ def genes(build, downloads_folder, api_key):
 
     # If required resources are missing, download them to a temporary file
     if downloads_folder is None:
-        with tempfile.TemporaryDirectory() as tempdir:
+        with tempfile.TemporaryDirectory(dir=os.getcwd()) as tempdir:
             try:
                 download_resources(tempdir, api_key, builds)
             except Exception as ex:
```

Comment:
Should execution continue if `download_resources()`? If I read the code correct it will quickly crash.

Reply:
I think it's fine. It will just exit with a warning, see line 90 in the code.

Reply:
That is basically what happens now before the bugfix